libr/core/cmd_open.c

```diff
@@ -1209,6 +1218,44 @@ static bool desc_list_json_cb(void *user, void *data, ut32 id) {
 	return true;
 }
 
+static bool cmd_op(RCore *core, char mode, int fd) {
+	int cur_fd = r_io_fd_get_current (core->io);
+	int next_fd = cur_fd;
+	switch (mode) {
+	case 0:
+		next_fd = fd;
+		break;
+	case 'n':
+		next_fd = r_io_fd_get_next (core->io, cur_fd);
+		break;
+	case 'p':
+		next_fd = r_io_fd_get_prev (core->io, cur_fd);
+		break;
+	case 'r':
+		next_fd = r_io_fd_get_next (core->io, cur_fd);
+		if (next_fd == -1) {
+			next_fd = r_io_fd_get_lowest (core->io);
+		}
+		break;
+	}
+	if (cur_fd != next_fd) {
```

Comment:
this check is already performed in the next line

Reply:
i'm not sure if i follow you. i dont see this check anywhere

Reply:
omg im blind. fixed